packages/uploadthing/test/client.test.ts

```diff
@@ -48,20 +48,24 @@ export const setupUTServer = (task: Test) => {
     }),
   );
 
-  const port = genPort(task);
-  const server = app.listen(port);
+  const server = app.listen(0);
+  const port = (server.address() as AddressInfo).port;
+  await new Promise((res) => server.once("listening", res));
```

Comment:
server was sometimes not started before the 1st test executed leading to some occasional `Fetch Failed`'s